dist/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="../dist/main.js"></script>
+  <title>Weather Broadcast</title>
+</head>
+
+<body>
+
+  <main>
+    <div class="container">
+
+      <div novalidate id="search-form">
+        <div>
+          <div class="select">
+            <input type="text" placeholder="Country Name">
+            <ul id="countries-list"></ul>
+          </div>
+          <div class="select">
+            <input type="text" placeholder="City Name" disabled>
+            <ul id="cities-list"></ul>
+          </div>
+        </div>
+        <button class="started">&#128270</button>
+      </div>
+
+      <div class="weather">
+        <div class="conversion"><span class="f">°F</span><label class="slider"></label><span class="c">°C</span><input
```

Comment:
consider hiding this section till we have a response [OPTIONAL]

Reply:
##### done :heavy_check_mark: